src/content/ui-components/install/laravel.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Laravel
+meta:
+  title: Laravel | Tiptap UI Components
+  description: Install and configure Tiptap UI Components in your Laravel project.
+  category: UI Components
+---
+
+Use the command below to scaffold a new Laravel project with React support:
+
+```bash
+laravel new my-app --react
+```
+
+## Add Tiptap components
+
+Install Tiptap UI components using the CLI. For example, to add the `HeadingButton` component:
+
+```bash
+npx @tiptap/cli add heading-button
+```
+
+The command above will install the `HeadingButton` component and its dependencies. You can then import and use it in your **Tiptap** project:
+
+```tsx
+import React from 'react'
+import { HeadingButton } from '@/components/ui/heading-button'
+
+export default function EditorPage() {
+  // Tiptap setup...
+
+  return (
+    <div className="editor-page">
+      <div className="toolbar">
+        <HeadingButton level={1}>Heading 1</HeadingButton>
+        <HeadingButton level={2}>Heading 2</HeadingButton>
+        <HeadingButton level={3}>Heading 3</HeadingButton>
+      </div>
+      {/* Your editor component */}
+    </div>
+  )
+}
+```
+
+## Add Styles
+
+Before proceeding, ensure that your project includes a CSS reset. If you're using Tailwind CSS, you can skip this step since it comes with a built-in reset.
+
+To enable support for SCSS in React App, install the SCSS compiler:
+
+```bash
+npm install -D sass-embedded
+```
+
+Then, to ensure the component and editor have the correct variables and animations, manually import the SCSS partials into your main stylesheet `src/index.css` or `src/App.css`:
+
+```css
+@import 'path-to/styles/_variables.scss';
+@import 'path-to/styles/_keyframe-animations.scss';
+```
```

Comment:
@Aslam97 Is this path correct? If not, how will developers know what path to use instead of `path-to`?